Common/DTO/User.cs

```diff
@@ -2,12 +2,51 @@
 
 namespace DTO
 {
-    public class User
+    public class User : IEquatable<User>
     {
         public Guid Id { get; set; }
         public string Email { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
         public DateTime Birthday { get; set; }
+
+        public bool Equals(User other)
```

Comment:
I think it's better to make checker in tests class, remove this code from data transfer objects please

Reply:
Then you will have `AreUsersEqual()` in tests and you will call `Assert.IsTrue()` for example